v23.1/transaction-retry-error-reference.md

```diff
@@ -97,13 +143,13 @@ The `RETRY_SERIALIZABLE` error occurs in the following cases:
 _Action_:
```

Comment:
>When a transaction A is forced to refresh (i.e., change its timestamp) due to hitting the maximum [closed timestamp](https://deploy-preview-16280--cockroachdb-docs.netlify.app/docs/v23.1/architecture/transaction-layer#closed-timestamps) interval (closed timestamps enable [Follower Reads](https://deploy-preview-16280--cockroachdb-docs.netlify.app/docs/v23.1/follower-reads#how-stale-follower-reads-work) and [Change Data Capture (CDC)](https://deploy-preview-16280--cockroachdb-docs.netlify.app/docs/v23.1/change-data-capture-overview)). This can happen when transaction A is a long-running transaction, and there is a write by another transaction to data that A has already read. If this is the cause of the error, the solution is to increase the kv.closed_timestamp.target_duration setting to a higher value. Unfortunately, there is no indication from this error code that a too-low closed timestamp setting is the issue. Therefore, you may need to rule out cases 1 and 2 (or experiment with increasing the closed timestamp interval, if that is possible for your application - see the note below).


This is actually good context to add to the shared section (Minimize transaction retry errors) above. If no other mitigations work, change the setting it this is a long-runing transaction